components/setup.sh

```diff
@@ -22,7 +22,18 @@ function setup() {
     bundle exec fastlane match appstore --readonly $verboseArg
   fi
 
+  msg 'Installing Git hooks'
+  symlinkGitHooks
+
   comp_deinit
 
   carthage_bootstrap
 }
+
+function symlinkGitHooks() {
+  hooksDir="$project_dir"/.git/hooks
+  mkdir -p "$hooksDir"
+  ln -s ../../bin/git-hooks/post-checkout "$hooksDir"
+  ln -s ../../bin/git-hooks/post-merge "$hooksDir"
```

Comment:
Maybe this would be clearer:

```bash
ln -s ../../bin/git-hooks/submodule-update $hooksDir/post-checkout
ln -s ../../bin/git-hooks/submodule-update $hooksDir/post-merge
```